content/en/docs/installation/openshift.md

```diff
@@ -64,10 +64,47 @@ Install the `CustomResourceDefinitions` and cert-manager itself
 oc apply -f https://github.com/jetstack/cert-manager/releases/download/v1.4.0/cert-manager.yaml
 ```
 
-## Installing with cert-manager operator
+## Installing with Operator Lifecycle Manager and OperatorHub Web Console
```

Comment:
praise: love the title casing :joy:

Reply:
What I'd like to know is how to give it a shorter / stable anchor name like `#olm`

Reply:
I just looked into it, looks like maybe it would work with HTML:

```html
<h2 id="olm">Installing with Operator Lifecycle Manager and OperatorHub Web Console</h2>
```